jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/ResultFixtures.java

```diff
@@ -36,7 +35,8 @@ public static Result booleanTrueResult() {
 				"ObjectDomainName",
 				"VerboseMemory",
 				"type=Memory",
-				ImmutableMap.<String, Object>of("Verbose", true));
+				ImmutableList.<String>of(),
```

Comment:
It would be nice if the valuePath was coherent with the rest of the parameters (I'm sure you intended to do the cleanup later...)

Reply:
What do you mean by "coherent with the rest of the parameters"

Reply:
Oh, my bad. In this case it is actually expected to have an empty valuePath.

Reply:
Yes one of the changes related to removing the Map is that in case of simple value we don't have to choose an arbitrary Map key.